From 13a930f8c22df65cff6ec3968729a33b365a854a Mon Sep 17 00:00:00 2001
From: Lukasz Bartosik <lbartosik@marvell.com>
Date: Fri, 28 Dec 2018 15:22:26 +0300
Subject: [PATCH 0876/1051] crypto: cpt - update dependencies in Kconfig

Update dependencies for OCTEONTX module in
drivers/net/ethernet/cavium/Kconfig.

Change-Id: I8b78a77b9eab07ac14c21346562722e2225e60bd
Signed-off-by: Lukasz Bartosik <lbartosik@marvell.com>
Reviewed-on: https://sj1git1.cavium.com/1783
Tested-by: sa_ip-sw-jenkins
Reviewed-by: Sunil Kovvuri Goutham <Sunil.Goutham@cavium.com>
[Kevin: The original patch got from Marvell sdk10.0_19.06]
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 drivers/net/ethernet/cavium/Kconfig | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/net/ethernet/cavium/Kconfig b/drivers/net/ethernet/cavium/Kconfig
index 2246028b85ae..9967761d3805 100644
--- a/drivers/net/ethernet/cavium/Kconfig
+++ b/drivers/net/ethernet/cavium/Kconfig
@@ -204,8 +204,8 @@ config OCTEONTX
 	depends on THUNDER_NIC_BGX && OCTEONTX_FPA_PF && OCTEONTX_SSO_PF
 	depends on OCTEONTX_PKO_PF && OCTEONTX_SSOW_PF && OCTEONTX_PKI
 	depends on OCTEONTX_DPI && OCTEONTX_LBK && OCTEONTX_TIM_PF
-	depends on OCTEONTX_ZIP_PF
-	depends on OCTEONTX_SLI_PF
+	depends on OCTEONTX_ZIP_PF && OCTEONTX_SLI_PF
+	depends on CRYPTO_DEV_OCTEONTX_CPT
 	default y
 	help
 	  Select this option to enable Octeon coprocessor management.
-- 
2.17.1

